ext/exif/exif.stub.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/** @return string|false */
+function exif_tagname(int $index) {}
+
+/** @return array|false */
+function exif_read_data($filename, ?string $sections_needed = null, bool $sub_arrays = false, bool $read_thumbnail = false) {}
+
+/** @return string|false */
+function exif_thumbnail($filename, &$width, &$height, &$imagetype) {}
```

Comment:
```suggestion
function exif_thumbnail($filename, &$width = null, &$height = null, &$imagetype = null) {}
```

These are all optional.

Reply:
I've added the default `null` to those now